### PR TITLE
Drone control icons

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[FEATURE]
## About the PR 
sorry pillars
Adds an icon over flockdrones that are being controlled by flockminds or flocktraces, only visible to flock

image   image

## Why's this needed? 
Good to know who is sentient

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
!merge_upstream

**Comment by amylizzle** [comment]
hows it look with flame's health icons also?

**Comment by TobleroneSwordfish** [comment]
> hows it look with flame's health icons also?

let me merg and find out

**Comment by TobleroneSwordfish** [comment]
image

Health icons don't render on the controller's client however, while the control icons do currently. Not sure which is better.

**Review by amylizzle — APPROVED** [review]

File: code/mob/living/critter/flock/flockdrone.dm [hunk 1]
```diff
@@ -139,6 +139,8 @@
 	controller = pilot
 	src.client?.color = null // stop being all fucked up and weird aaaagh
 	src.hud?.update_intent()
+	var/image/icon = flock.add_overhead_image('icons/misc/featherzone.dmi', src, pilot.control_icon)
```
Comment (amylizzle) [inline code/mob/living/critter/flock/flockdrone.dm:142]: Oh actually maybe this should be in a proc on the flock datum

File: code/mob/living/critter/flock/flockdrone.dm [hunk 2]
```diff
@@ -169,6 +171,9 @@
 				controller.mind.key = key
 				controller.mind.current = controller
 				ticker.minds += controller.mind
+		var/image/I = flock.annotations_control_icons[src]
```
Comment (amylizzle) [inline code/mob/living/critter/flock/flockdrone.dm:174]: this too

**Review by amylizzle — CHANGES_REQUESTED** [review]
proc up some things

**Pull request merged**